packages/dd-trace/src/appsec/api_security_sampler.js

```diff
@@ -70,7 +65,7 @@ function isSampled (req, res) {
 function computeKey (req, res) {
   const route = req.route?.path || req.url
   const method = req.method.toLowerCase()
-  const statusCode = res.statusCode
+  const statusCode = res.statusCode === 304 ? 200 : res.statusCode
```

Comment:
this is funny because sometimes, for the same request, status code is `200` in `onResponseBody` but `304` in `incomingHttpEndTranslator`. It depends on when you are trying to get it.